readme/Readme.scalatex

```diff
@@ -322,6 +335,89 @@
+            but, naturally, it's recommended to follow the one uniform style and @hl.scala{BS} with @hl.scala{strToBytes} is the good example of it.
+
+        @sect{New Parsers}
+            @p
+                Byte Parsers doesn't bring anything new to the basic set of parsers.
+                @hl.scala{Option}, @hl.scala{Repeat}, @hl.scala{Capture} and ect. are still here with the same logic,
+                but there are some new names for @hl.scala{Byte} versions of some of them.
+                It just a replacement of "Char" substring to the "Byte", that's all, so
+                @ul
+                    @li
+                        @hl.scala{AnyChar} becomes @hl.scala{AnyByte}
+                     @li
+                        @hl.scala{CharPred} becomes @hl.scala{BytePred}
+                     @li
+                        @hl.scala{CharIn} becomes @hl.scala{ByteIn}
+                     @li
+                        @hl.scala{CharsWhile} becomes @hl.scala{BytesWhile}
+                     @li
+                        @p
+                            @hl.scala{StringIn} becomes @hl.scala{SeqIn}
+                        @i
+                            It isn't a simple replacement of substrings, and the reason for this is described in @sect.ref{Inner Structure}.
+
+             @p
+                For instance, the basic parser with @hl.scala{AnyByte}
+             @hl.ref(bytetests/"ByteTests.scala", Seq("'anychar", ""))
+
+    @sect{Inner Structure}
+        @p
+            Actually, each of the previously described parsers is a special case of more abstract
+            corresponding parser from the inner Api that can be used for the arbitrary type of elements in the input.
+            Parsers for @hl.scala{String} input has @hl.scala{Char} as type-parameter in this Api,
+            parsers for @hl.scala{Array[Byte]} has @hl.scala{Byte} and etc.
+        @p
+            There is no need to explain again how these abstract parsers work,
+            because it's exactly the same as in the @hl.scala{String} or @hl.scala{Array[Byte]} case,
+            the only difference is that instead of @hl.scala{Char} or @hl.scala{Byte} we consider arbitrary class.
+            Names of these parsers are also the same except of
+            @ul
+                @li
+                    @hl.scala{AnyElem} for @hl.scala{AnyChar} and @hl.scala{AnyByte}
+                @li
+                    @hl.scala{ElemPred} for @hl.scala{CharPred} and @hl.scala{BytePred}
+                @li
+                    @hl.scala{ElemIn} for @hl.scala{CharIn} and @hl.scala{ByteIn}
+                @li
+                    @hl.scala{ElemsWhile} for @hl.scala{CharsWhile} and @hl.scala{BytesWhile}
+                @li
+                    @hl.scala{SeqIn} for @hl.scala{StringIn}
+            but you can, of course, use these abstract versions of names wherever you want.
+
+    @sect{Iterator Parsers}
```

Comment:
Let's call this `Iterator Parsing` instead of `Iterator Parsers`, because this applies to all existing and future parsers, not just those specifically written to work with iterators.